File: 01_IFSP/LP1/Lista_3/Exercicios_CASA-lista-3/EX10-Operacoes-c/EX10-c.c
#include <stdio.h>

int main(){
    char esc;
    int n1, n2;
    esc='e';
    while(esc != 'p'){
        scanf(" %c", &esc);
        if(esc == '+'){
            scanf("%d", &n1);
            scanf("%d", &n2);
            printf("%d\n", n1+n2 );
        }
        else if(esc == '-'){
            scanf("%d", &n1);
            scanf("%d", &n2);
            printf("%d\n", n1-n2 );
        }
        else if(esc == '*'){
            scanf("%d", &n1);
            scanf("%d", &n2);
            printf("%d\n", n1*n2 );
        }
        else if(esc == '/'){
            scanf("%d", &n1);
            scanf("%d", &n2);
            printf("%f\n", (float)n1/n2 );
        }
        else if (esc != 'p'){
            printf("Opcao Invalida\n");
        }
    }
    return 0;
}

File: 01_IFSP/LP1/Lista_3/Exercicios_CASA-lista-3/EX10-Operacoes-c/EX10-c.py
while (escolha:=input().upper())!= "P":
    if escolha == "+":
        print(int(input())+int(input()))
    elif escolha == "-":
        print(int(input())-int(input()))
    elif escolha =="*":
        print(int(input())*int(input()))
    elif escolha =="/":
        print(int(input())/int(input()))
    else:
        print("Opcao Invalida")